grafana-tikv-dashboard.md

```diff
@@ -484,6 +484,44 @@ This section provides a detailed description of these key metrics on the **TiKV-
 - Initial Scanning Trigger Reason: The reason for triggering incremental scanning
 - Region Checkpoint Key Putting: The number of checkpoint operations logged to the PD
 
+### Import & Snapshot restore
+
+- Import CPU Utilization: CPU utilization aggregated by sst importer.
+- Import Thread Count: number of threads used by sst importer.
+- Import Errors: error counts during sst import.
+- Import RPC Duration: time spent for various RPC calls in sst importer.
+- Import RPC Ops: number of total RPC calls in sst importer.
+- Import RPC Count: number of inflight RPC calls in sst importer.
+- Import Write/Download RPC Duration: RPC time for write/download in sst importer.
+- Import Wait Duration: time spent on downloading task waiting in queue for execution.
+- Import Read SST Duration: time spent on reading from external storage of a file and download it to TiKV.
+- Import Rewrite SST Duration: time spent on rewriting SST based on rewrite rules. 
+- Import Ingest RPC Duration: time spent on handling ingest RPC request on TiKV.
+- Import Ingest SST Duration: time spent on ingesting SST into RocksDB.
+- Import Ingest SST Bytes: number of bytes ingested.
+- Import Download SST Throughput: SST download throughput in bytes per second.
+- cloud request: number of request to cloud providers. 
+
+### Point-in-Time Restore
+
+- CPU Usage: CPU utilization by PITR.
+- P99 RPC Duration: 99 percentile of the RPC requests time.
+- Import RPC Ops: number of total RPC calls in sst importer.
+- Import RPC Count: number of inflight RPC calls in sst importer.
+- Cache Events: number of events on file cache during sst import.
+- Overall RPC Duration: time spent on RPC calls.
+- Read File into Memory Duration: time spent on downloading files from external storage and loaded in to memory.
+- Queuing Time: time spent on waiting to get scheduled on a thread.
+- Apply Request Throughput: Apply request rate in bytes.
+- Downloaded File Size: downloaded file size in bytes.
+- Apply Batch Size: number of bytes for applying to Raft store in one batch. 
+- Blocked by Concurrency Time: time spent on waiting to get executed due to concurrency constraint.  
+- Apply Request Speed: speed of applying request to Raft store. 
+- Cached File in Memory: files cached by the applying requests of importer.
+- Engine Requests Unfinished: number of pending requests to Raft store.
+- Apply Time: time spent on writing data to the Raft store.
+- Raft Store Memory Usage: memory usage for Raft store.
```

Comment:
```suggestion
- CPU Usage: The CPU utilization by point-in-time recovery (PITR).
- P99 RPC Duration: The 99th percentile of RPC request time.
- Import RPC Ops: The total number of RPC calls in SST importer.
- Import RPC Count: The number of inf-light RPC calls in SST importer.
- Cache Events: The number of events on file cache during SST import.
- Overall RPC Duration: The time spent on RPC calls.
- Read File into Memory Duration: The time spent downloading files from external storage and loading them into memory.
- Queuing Time: The time spent waiting to be scheduled on a thread.
- Apply Request Throughput: The rate of applying requests in bytes.
- Downloaded File Size: The size of downloaded file in bytes.
- Apply Batch Size: The number of bytes for applying to Raft store in one batch.
- Blocked by Concurrency Time: The time spent waiting for execution due to concurrency constraints.
- Apply Request Speed: The speed of applying request to Raft store.
- Cached File in Memory: The files cached by the applying requests of importer.
- Engine Requests Unfinished: The number of pending requests to Raft store.
- Apply Time: The time spent writing data to the Raft store.
- Raft Store Memory Usage: The memory usage for Raft store.
```

Reply:
Thanks for the change! Somehow I don't have the permission to commit from web UI, can only do it offline from my local change